plugins/com.google.cloud.tools.eclipse.bugreport/plugin.properties

```diff
@@ -0,0 +1 @@
+bugReportMenu=Report Cloud Tools for Eclipse bug...
```

Comment:
I agree, the shorter looks better. If we customize the link we could provide 2 menu items one for bugs one for feature requests and the issues created would have the proper label (bug/enhancement) attached.